WordPressPCL/WordPressClient.cs

```diff
@@ -186,8 +187,18 @@ public async Task RequestJWToken(string Username, string Password)
                     new KeyValuePair<string, string>("username", Username),
                     new KeyValuePair<string, string>("password", Password)
                 });
-            (JWTUser jwtUser, _) = await _httpHelper.PostRequest<JWTUser>(route, formContent, false).ConfigureAwait(false);
-            _httpHelper.JWToken = jwtUser?.Token;
+            if (AuthMethod == AuthMethod.JWT)
+            {
+                (JWTUser jwtUser, _) = await _httpHelper.PostRequest<JWTUser>(route, formContent, false).ConfigureAwait(false);
+                _httpHelper.JWToken = jwtUser?.Token;
+            }
+            else
```

Comment:
I'd prefer checking here explicitly for `AuthMethod == AuthMethod.JWTAuth` to make it more verbose in case other methods or plugins are added in the future.

Reply:
I adapted it. Thanks for your input